client/src/components/App.js

```diff
@@ -30,10 +31,11 @@ const App = (props) => {
     <Router>
       <TopBar user={currentUser} />
       <Switch>
-        <Route exact path="/">
-          <h2>Hello from react</h2>
-        </Route>
-        <Route exact path="/restaurants" component={RestaurantList} />
+        <Route exact path="/" component={HomePage} 
+        />
+        <Route 
+        exact path="/restaurants" component={RestaurantList} 
+        />
```

Comment:
And this should be `<Route exact path="/restaurants" component={RestaurantList} />`